pkg/sink/cloudstorage/table_definition_test.go

```diff
@@ -445,3 +445,7 @@ func TestTableDefinition(t *testing.T) {
 	require.Equal(t, timodel.ActionAddColumn, event.Type)
 	require.Equal(t, uint64(100), event.CommitTs)
 }
+
+func TestTableDefinitionSum32(t *testing.T) {
+	// TODO: add sum32 tests
```

Comment:
Maybe we can add it. I guess it is easy to add. At least, we should use `skip()` to ship this unimplemented case.

Reply:
https://github.com/pingcap/tiflow/issues/8914